### PR TITLE
Test KubeVirt v0.8.0-alpha.1

### PR DESCRIPTION
**Pull request by gbenhaim**
Signed-off-by: gbenhaim <galbh2@gmail.com>

### CODE REVIEW

File: roles/kubevirt/defaults/main.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ cluster: "openshift"
 apb_action: "provision"
 release_manifest_url: "https://github.com/kubevirt/kubevirt/releases/download"
 kubevirt_template_dir: "{{ role_path }}/templates"
-version: 0.7.0
+version: v0.8.0-alpha.1
```
Comment (lukas-bednar) [inline roles/kubevirt/defaults/main.yml]: there is extra `v`

File: vars/all.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ openshift_ansible_dir: "openshift-ansible/"
 openshift_playbook_path: "{{ openshift_ansible_dir }}/{{ 'playbooks/byo/config.yml' if kubevirt_openshift_version == '3.7' else 'playbooks/deploy_cluster.yml' }}"
 
 ### KubeVirt ###
-version: 0.7.0
+version: v0.8.0-alpha.1
```
Comment (lukas-bednar) [inline vars/all.yml]: extra `v` here too

**Comment by gbenhaim** [comment]
@lukas-bednar 
I'm getting the following error when trying to update the vendor dir: 
```
ERROR]	Error scanning k8s.io/apimachinery/pkg/apimachinery/announced: open /home/gbenhaim/.glide/cache/src/https-k8s.io-apimachinery/pkg/apimachinery/announced: no such file or directory
[ERROR]	This error means the referenced package was not found.
[ERROR]	Missing file or directory errors usually occur when multiple packages
[ERROR]	share a common dependency and the first reference encountered by the scanner
[ERROR]	sets the version to one that does not contain a subpackage needed required
[ERROR]	by another package that uses the shared dependency. Try setting a
[ERROR]	version in your glide.yaml that works for all packages that share this
[ERROR]	dependency.
[ERROR]	Error scanning k8s.io/apimachinery/pkg/apimachinery/registered: open /home/gbenhaim/.glide/cache/src/https-k8s.io-apimachinery/pkg/apimachinery/registered: no such file or directory
```

**Comment by lukas-bednar** [comment]
@gbenhaim you have most probably misconfigured your environment ...
It works for me with
```
[lbednar@lbednar kubevirt-ansible]$ go version 
go version go1.10.3 linux/amd64
[lbednar@lbednar kubevirt-ansible]$ glide --version
glide version 0.13.2-dev
```
https://paste.fedoraproject.org/paste/lBQkxpBp-juopombLNp3YA

**Comment by gbenhaim** [comment]
@lukas-bednar `tests/util.go:87:14: undefined: tests.RunOcCommand`
It was disappeared from `vendor/kubevirt.io/kubevirt/tests/utils.go`, but still exist in kubevirt repo.
Any idea?

**Comment by lukas-bednar** [comment]
@cynepco3hahue could you please see what has happened with `tests.RunIOcCommnad` ? https://github.com/kubevirt/kubevirt-ansible/pull/367#issuecomment-417006708

**Comment by shiywang** [comment]
@gbenhaim  `undefined: tests.RunOcCommand` was removed in upstream, the only change to make it pass I think is here: https://github.com/kubevirt/kubevirt-ansible/pull/354/files#diff-8337e1399854c6a050cd98f558f9ce68R162 and here: https://github.com/kubevirt/kubevirt-ansible/pull/354/files#diff-8337e1399854c6a050cd98f558f9ce68R167

**Comment by gbenhaim** [comment]
@shiywang I still see it under the `v0.8.0-alpha.1`.
```
 gbenhaim ~/go/src/kubevirt.io/kubevirt (master)
 > git checkout v0.8.0-alpha.1
warning: refname 'v0.8.0-alpha.1' is ambiguous.
Switched to branch 'v0.8.0-alpha.1'
 gbenhaim ~/go/src/kubevirt.io/kubevirt (v0.8.0-alpha.1)
 > git grep RunOcCommand
tests/template_test.go: out, err := tests.RunOcCommand(args...)
tests/template_test.go: out, err := tests.RunOcCommand("create", "-f", vmJsonFile)
tests/template_test.go: out, err := tests.RunOcCommand("patch", "virtualmachine", vmName, "--type", "merge", "-p", patch)
tests/template_test.go: out, err := tests.RunOcCommand("delete", "vm", vmName)
tests/template_test.go: out, err := tests.RunOcCommand("get", resourceType)
tests/utils.go:func RunOcCommand(args ...string) (string, error) {

```

**Comment by gbenhaim** [comment]
@shiywang My mistake, the tags in my local copy wasn't correct.

**Comment by gbenhaim** [comment]
@shiywang Any ETA on the fix?

**Comment by shiywang** [comment]
@gbenhaim https://github.com/gbenhaim/kubevirt-ansible/pull/2

**Pull request merged**